plotting/map.py

```diff
@@ -954,6 +982,4 @@ def find_lines(values):
                 fontsize=14,
             )
 
-        fig.tight_layout(pad=3, w_pad=4)
-
         return super(MapPlotter, self).plot(fig)
```

Comment:
This piece is incredibly slow. It's currently taking 70+ seconds to get through the `plt.savefig()` function in plotter.py. I haven't been able to figure this out yet. Any suggestions @htmlboss?

Reply:
Seems to be a known issue....any chance we can use `plt.imsave` instead of `savefig`? https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.imsave.html Apparently it could be faster...

In other news, matplotlib supports interactive plots in the browser via websockets if you change the plotting backend LOL

Reply:
Setting the rendering style to fast could help too? https://matplotlib.org/stable/users/explain/performance.html#using-the-fast-style

Reply:
For reference: https://github.com/matplotlib/matplotlib/issues/16758

Reply:
Thanks! Looks like there's a few ways forward I can try.

Reply:
I didn't have much luck on the matplotlib side of things. Looks like `plt.imsave` saves arrays to images which won't really work for us here and I didn't see any improvement when setting the rendering style to fast. 

I'm thinking this might be an issue with how Cartopy handles various projections. If we change the projection from LCC to Plate Carree (the default for Cartopy) I can get the savefig time down to 12 s, more or less in line with what we were getting with basemap. The plot generation will also be much quicker. The catch is that it's an equirectangular projection so there'll be some distortion in the resulting maps. 

Here's the plot from above using Plate Carree projection:
![pc_plot](https://user-images.githubusercontent.com/79917349/164716345-fa727718-daeb-4c54-b80e-95ce5d2e51e8.png)